models/dalec/DESCRIPTION

```diff
@@ -5,7 +5,8 @@ Version: 1.7.3.9000
 Authors@R: c(person("Mike", "Dietze", role = c("aut", "cre"),
                      email = "dietze@bu.edu"),
              person("Tristan", "Quaife", role = c("aut")),
-             person("University of Illinois, NCSA", role = c("cph")))
+             person("University of Illinois, NCSA", role = c("cph")),
+             person("Boston University", role = c("cph")))
```

Comment:
This is the only package so far where I have edited copyright holders in DESCRIPTION, because it's the only one where I deleted headers from R files that listed anyone other than UI/NCSA as a copyright holder. 

As a separate project that _doesn't_ need to tag everyone a zillion times, we could consider going back through to (1) check whether BU / UA / others need to be identified by name as copyright holders of other packages, and (2) update years as needed to indicate the full range of edit dates.

Reply:
I'd recommend dropping the role rather than adding it elsewhere or expanding it to other universities or institutions. Blame says @robkooper added this here https://github.com/PecanProject/pecan/commit/b0dd245365bfcb1b47ab5b620fcfc82348a961c8 3 years ago, which is well after all of us except Rob left UIUC and I don't recall us collectively agreeing to give UIUC or NCSA copywrite permissions to the code. Seems like that should be granted to the PEcAn project, same as the licence